app/src/test/kotlin/me/myshows/android/api/jsonrpc/JsonRPCTest.kt

```diff
@@ -0,0 +1,81 @@
+package me.myshows.android.api.jsonrpc
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import me.myshows.android.api.BaseMockWebServerTest
+import okhttp3.mockwebserver.MockResponse
+import okhttp3.mockwebserver.RecordedRequest
+import okio.Buffer
+import org.assertj.core.api.Java6Assertions.assertThat
+import org.junit.Before
+import org.junit.Test
+import retrofit2.Call
+import retrofit2.Retrofit
+import retrofit2.http.Body
+import retrofit2.http.POST
+
+class JsonRPCTest : BaseMockWebServerTest() {
+
+    private val mapper: ObjectMapper = jacksonObjectMapper()
```

Comment:
Why do we need explicit type here?

Reply:
Why not?
If you change type of init expression you'll break only this line, otherwise you have good chance to make all this file red in IDE.
So I try write explicit type of all not local variables.